pom.xml

```diff
@@ -104,7 +104,91 @@
             </excludes>
           </configuration>
         </plugin>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-release-plugin</artifactId>
+          <version>2.5.3</version>
+          <configuration>
+            <mavenExecutorId>forked-path</mavenExecutorId>
+            <useReleaseProfile>false</useReleaseProfile>
+            <arguments>${arguments}</arguments>
```

Comment:
we can add `-Dmaven.deploy.skip=true -Dgpg.skip=true` here if we don't want to specify these 2 props in the CLI